ui/app/routes/vault/cluster/clients/counts/sync.ts

```diff
@@ -4,5 +4,36 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+import { DEBUG } from '@glimmer/env';
 
-export default class ClientsCountsSyncRoute extends Route {}
+import type StoreService from 'vault/services/store';
+
+interface ActivationFlagsResponse {
+  data: {
+    activated: Array<string>;
+    unactivated: Array<string>;
+  };
+}
+
+export default class ClientsCountsSyncRoute extends Route {
+  @service declare readonly store: StoreService;
+
+  async getActivatedFeatures() {
+    try {
+      const resp: ActivationFlagsResponse = await this.store
+        .adapterFor('application')
+        .ajax('/v1/sys/activation-flags', 'GET');
+      return resp.data?.activated;
+    } catch (error) {
+      if (DEBUG) console.error(error); // eslint-disable-line no-console
```

Comment:
🗒️ pro tip -- there will probably be a better way to do this with Ember 5, but this is a a handy way to run code in non-production environments only. i like it for `console.log`ing errors that we want for local development / test debugging but that we'd otherwise want to hide/abstract from users.

see more [details in this blog](https://www.rwjblue.com/2017/05/01/enabling-debug-code-stripping/)